minio_service/test_client_minio.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from minio_client import *
```

Comment:
`import *` might creat a lot of problems, please import only what you need

Reply:
added

Reply:
I still see this import